test/regression.uts

```diff
@@ -1217,7 +1217,7 @@ in6_addrtomac("FE80::" + in6_mactoifaceid("FF:00:00:00:00:00", ulbit=0)) == 'ff:
 import socket
 
 res=True
-for _ in xrange(10):
+for a in xrange(10):
```

Comment:
ok and you need `_` after that I get it. Normally that's not a problem but `_` is actually used in the tests!